distribution/test/itests/test-itests-common/src/main/resources/TestSpatial/csw/request/query/CswPagingTestLikeQuery.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!-- /**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/ -->
+<csw:GetRecords
+        xmlns:ogc="http://www.opengis.net/ogc"
+        xmlns:csw="http://www.opengis.net/cat/csw/2.0.2"
+        resultType="results"
+        outputFormat="application/xml" outputSchema="http://www.opengis.net/cat/csw/2.0.2"
+        startPosition="1" maxRecords="20" service="CSW" version="2.0.2">
```

Comment:
Here we are specifying we only want 20 results - but in the test we verify all 30 are returned - I believe this confirms the `maxRecords` is incorrectly being ignored.